### ISSUE TITLE
audio play() fail at ios 12

### ISSUE THREAD

**Issue by 1097364388** [mrdoob/three.js#15566]
##### Description of the problem
After my test, due to the addition of this line of code, the Google browser audio playback in ios (12.1) failed.(经过我测试,由于添加了这行代码,使得在ios(12.1) 的谷歌浏览器音频播放失败.)
file path is: `src/audio/Audio.js`
```js
source.detune.value = this.detune;
```

##### Three.js version

- [ ] Dev
- [x] r100
- [ ] ...

##### Browser

- [ ] All of them
- [x] Chrome for iOS
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [] All of them
- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [x] iOS

**Comment by Mugen87** [comment]
Unfortunately, I can't verify this since I have no iOS device. Maybe someone else with an iPhone can test with the Web Audio examples:

https://threejs.org/examples/?q=webaudio

**Comment by harmvandeven** [comment]
@Mugen87 I've tested the examples on iOS both Safari and Chrome. The **Timing** example doesn't work, the others do. I ran into this thread because of an error I'm experiencing on _Safari macOS (12.0.2 on Mojave)_ as well. It is the same **Timing** example that doesn't work.

In both iOS and macOS it returns the same error: `TypeError: undefined is not an object (evaluating 'source.detune.value = this.detune')`
